### DISCUSSION TITLE
Avoid "Exiting due to MK_USAGE_NO_PROFILE: No minikube profile was found." On using minikube profile list [Colima 0.5.4]

### DISCUSSION THREAD

**Discussion by gdorsi44** [abiosoft/colima#673] [Ideas]
Hello ,

is it possibile to attach a minikube's profile when I getting started a colima default configuration as follows:

`colima start`

A sort of string attached like this
`colima start --kubernetes --profile myprofile1`

**Comment by abiosoft** [comment]
I do not know what you mean by minikube profile. But if you are referring to having multiple instances of Colima, then yes Colima does have an optional `--profile` flag that can be used for that.

The following commands do the same thing.

```
colima start myprofile1
colima start --profile myprofile1
```

**Comment by gdorsi44** [comment]
Thank you for the reply.
Minikube profile is the kubernetes' profile when I launch the command on Minikube program:
